### PR TITLE
[IMP] Inventory: Update inter-warehouse replenishment

### PR DESCRIPTION
**Pull request by hojo-odoo**
Task ID # [2812664](https://www.odoo.com/web#id=2812664&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

This PR implements five key changes:

- Removes the "Inter-warehouse transfers" document
- Moves the "Resupply from another Warehouse" document from the "Inventory Adjustment" folder to the "Warehouses" folder
- Deletes the now-empty "Inventory Adjustment" folder
- Updates the content of the "Resupply from another Warehouse" document and renames it to "Transfer products between warehouses using replenishment"
- Redirects the links for the "Inter-warehouse transfers" and "Resupply from another Warehouse" documents to the "Transfer products between warehouses using replenishment" document

### CODE REVIEW

**Comment by StraubCreative** [comment]
@odoo/us-doc-review this doc on warehouse transfers is ready for your review 👀 
Tag @samueljlieber for technical review when complete, thanks!

**Comment by StraubCreative** [comment]
@samueljlieber can you review this asap please? It's for PO task so let's get it merged by EOD Friday 🙏

File: content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst:6]: Let's italicize _Odoo Inventory_ here because it is an app name.

File: content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst:45]: :menuselection:Inventory

File: content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst:83]: :guilabel:Delivery Orders

File: content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst:95]: double 'the'

File: content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst:103]: image attribute alignment

File: content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst:130]: image attribute alignment

File: content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst:134]: :menuselection:Inventory

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @hojo-odoo 👋 Great job with the restructuring of these docs. I've made some technical changes that you can review below. I will push these changes up as a second commit for your approval. Once approved I will squash the commits, rebase, and push back up for final review.

Here is a breakdown of the changes:
- italicize app names
- use :menuselection: on an app name when directing the user instead of :guilabel:. 
   > Starting in :menuselection:Inventory select...
- use :guilabel: on UI elements when directing the user in an instructional sense
   > ...on the :guilabel:Delivery Orders card for...
- alignment of image attributes and admonition block text

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @hojo-odoo 

Great doc! It's clear you put in the time/research around a complex topic and I think you did a great job synthesizing it in a clear/simple way. Nice work 💪. 

For the minimum CRs to address before merge— these are very small technical things, as follows (with more details below in individual comments).
- italicize just the app name + specific settings
- small wording improvements, for clarity
- confirm **Check Availability** button
- menuselection use

Everything else is optional and for your consideration now or on a future PR.

**Next steps:**
I'm pushing up another commit in just a moment that has already addressed the minimum requirements to merge. You can pull these to your computer (`git pull --rebase`) and they should be visible right when you open VSC or run Make. However, if not, you can also `git checkout <commit>` to view the commit specifically.

- **if you approve of the changes I made**, go ahead and squash the commits, rebase and push back up (or I can do it just lmk). Then, let the CI checks run, and once they pass, issue the merge command (`[at]robodoo r+`) since I'm delegating it to you.
- if you do not approve of the changes, you can just apply w/e changes you'd prefer on a third commit, and then when you squash it will overwrite what I did. You can also completely reset the branch to it's original state before my second commit using the `git reset --HARD <commig>` command. In either case, squash your changes, rebase, and push back up to overwrite what's here on GH. After which, let the CI checks run and issue the merge command when they pass.

Thanks and let me know if you have any questions!
@robodoo delegate+

**Review by StraubCreative — APPROVED** [review]
Approving so you can run the merge command after you approve/improve using instructions above 🤙

File: content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
Comment (hojo-odoo) [reply content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst:89]: I think this may be because you had two- or three-step routes turned on for the outgoing warehouse. I just ran through it again in runbot and the "Check Availability" button appears.

File: content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
Comment (hojo-odoo) [reply content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst:93]: I'm going to leave this as-is just because the Inventory overview page can look different depending on a few factors: the number of warehouses created, the multi-step routes settings for those warehouses, etc. We're already telling the user what page to look for the card on and what the card will look like. If we show them a picture of the full UI, they could get confused because their "Delivery Orders" card appears in a different place than the one in the example.

**Comment by hojo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
